projectmanagement/src/main/java/life/qbic/projectmanagement/application/OfferSearchService.java

```diff
@@ -0,0 +1,46 @@
+package life.qbic.projectmanagement.application;
+
+import java.util.List;
+import java.util.stream.Collectors;
+import life.qbic.projectmanagement.domain.offer.OfferId;
+import life.qbic.projectmanagement.domain.project.ProjectTitle;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Service;
+
+/**
+ * <b>Offer Search Service</b>
+ *
+ * Enables search queries for offers.
+ *
+ * @since 1.0.0
+ */
+@Service
+public class OfferSearchService {
```

Comment:
I expected this Service to be the ProjectCreationService. This service then uses the offersearchservice to query for an offer. For me its not so intuitive that we have two OfferSearchServices. Why do we need a service that calls an equally called service from another layer?

Reply:
Ah yes, that makes a lot of sense. I was struggling with finding the proper class name.